FrontEnd/src/components/SignUp/UI/dotDecor/DotDecor.jsx

```diff
@@ -13,15 +15,17 @@ const DotRow = () => {
     </svg>
   );
 
-  const dots = [...Array(7)].map(() => circle);
+  const dots = [...Array(7)].map((_, index) => (
+    <React.Fragment key={index}>{circle}</React.Fragment>
+  ));
 
   return <div className={css['dot-row']}>{dots}</div>;
 };
 
 function DotDecorComponent(props) {
-  const blockPosition = props.position;
+  const { position } = props;
```

Comment:
You can destructure props directly in function arguments.
` function DotDecorComponent({ position }) {`

Reply:
Done